package.json

```diff
@@ -20,8 +20,9 @@
     "test": "npm run lint && npm run test-unit",
     "test-integration": "npm run start-test & wait-on http://localhost:8090 && npx cypress run",
     "clean": "del-cli $npm_package_config_dist_root/* ./config/assets.json",
-    "build-sass": "node-sass assets/sass --output $npm_package_config_dist_css --source-map $npm_package_config_dist_css",
-    "build-css": "npm run build-sass && postcss $npm_package_config_dist_css/*.css --replace --map",
+    "build-css-style": "node-sass assets/sass/style.scss --output $npm_package_config_dist_css --source-map $npm_package_config_dist_css && postcss $npm_package_config_dist_css/style.css --replace --map",
+    "build-css-emails": "node-sass assets/sass/emails.scss | postcss --no-map -u autoprefixer > $npm_package_config_dist_css/emails.css",
```

Comment:
The `-u` overrides the config and tells postcss to just use autoprefixer, `--no-map` is needed to disable the default inline sourcemap behaviour. Because we are processing a single file we can pipe the output rather than needing to write files temporarily and reprocess them like we do with the main stylesheet.